src/utils/string/StringUtils.js

```diff
@@ -163,6 +163,7 @@ const displayFormattedSearchLocationsV2 = (searchQuery, locationObject) => {
   return null;
 };
 
+// eslint-disable-next-line consistent-return
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `eslint-disable-next-line consistent-return` suggests that there might be inconsistent return points in the function `getChatName`. Ensure that all code paths in a function return a value.
```diff
- // eslint-disable-next-line consistent-return
  const getChatName = (usernames, me) => {
    if (!usernames) {
      return 'No Name';
    }
+   // Rest of the function...
+   return 'No Name'; // Ensure a default return at the end
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->